### PR TITLE
Homework5

### PR DESCRIPTION
**Pull request by Jalmer13**

### CODE REVIEW

File: Homework5/task1.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package Homework5;
+public class task1 {
```
Comment (AndrewSkoba) [inline Homework5/task1.java]: Pls, add empty row between package and class

File: Homework5/task1.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    public static void main(String[] args) {
+        String str1 = "Tenet";
+        String str2 = str1.toLowerCase();
+        char[] result = str2.toCharArray();
+        int l = result.length;
```
Comment (AndrewSkoba) [inline Homework5/task1.java]: Name more meaningful

File: Homework5/task1.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        boolean pal = true;
```
Comment (AndrewSkoba) [inline Homework5/task1.java]: same

File: Homework5/task3.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package Homework5;
+import java.util.Random;
+public class task3 {
+    public static void main(String[] args) {
+        int[] arrayTrd = new int[10];
+        Random rnd = new Random();
+        for(int i = 0; i < arrayTrd.length; ++i) {
```
Comment (AndrewSkoba) [inline Homework5/task3.java]: In a lecture was some mistake. For a loop statement there is no deference in i++ and ++i
So, both of them just increase i after each iteration

File: Homework5/task1.java (hunk 1 above)
Comment (AndrewSkoba) [inline Homework5/task1.java]: Class name from capital letter

Comment (Jalmer13) [reply Homework5/task1.java]: Done

File: Homework5/task1.java (hunk 2 above)
Comment (Jalmer13) [reply Homework5/task1.java]: int lenght = result.length

File: Homework5/task1.java (hunk 3 above)
Comment (Jalmer13) [reply Homework5/task1.java]: boolean palindrome = true

File: Homework5/task3.java (hunk 4 above)
Comment (Jalmer13) [reply Homework5/task3.java]: done

File: Homework5/task1.java (hunk 1 above)
Comment (Jalmer13) [reply Homework5/task1.java]: done

**Review by AndrewSkoba — APPROVED** [review]

**Pull request merged**